packages/@lightningjs/ui-components/src/components/CardContent/CardContentVerticalSmall.stories.js

```diff
@@ -49,22 +48,22 @@ export const CardContentVerticalSmall = args =>
   };
 
 CardContentVerticalSmall.storyName = 'CardContentVerticalSmall';
+const cardContentArgs = CardContent.args;
+delete cardContentArgs.metadata_details;
```

Comment:
everything seems fine, but does it matter that this technically mutates the object? Maybe line 51 should be an Object.assign or JSON.parse(JSON.stringify?

Reply:
Yeah, How about something like this:
```
const cardContentArgs = { ...CardContent.args };
delete cardContentArgs.metadata_details;
```